jax/_src/api.py

```diff
@@ -1403,7 +1403,7 @@ def vmap(fun: F, in_axes=0, out_axes=0, axis_name=None, axis_size=None) -> F:
       range ``[-ndim, ndim)`` for each array, where ``ndim`` is the number of
       dimensions (axes) of the corresponding input array.
 
-      If the positional arguments to ``fun`` are container types, the
+      If the positional arguments to ``fun`` are container (pytree) types, the
       corresponding element of ``in_axes`` can itself be a matching container,
       so that distinct array axes can be mapped for different container
       elements. ``in_axes`` must be a container tree prefix of the positional
```

Comment:
Since the mechanism of parameters that are container tree prefix is used in several places in the API, I think it is worth consolidating a more in-depth explanation and linking to it. E.g., [this](https://jax.readthedocs.io/en/latest/pytrees.html?highlight=arguments#applying-optional-parameters-to-pytrees) in the pytree docs.

Reply:
Great idea!